app/init/global_event_handler.js

```diff
@@ -213,6 +214,7 @@ class GlobalEventHandler {
 
         await dispatch(loadConfigAndLicense());
         await dispatch(loadMe(user));
+        dispatch(getTeams());
```

Comment:
We are awaiting the rest of the dispatches in this function. Any reason not to wait for this one?

Reply:
The other two are sort of required when the rest of the UI loads, the last one is just something that we really don't depend on